docs/docs/reference/reference/smart_contract_reference/aztec-nr/address-note/address_note.md

```diff
@@ -0,0 +1,61 @@
+# AddressNote
+
+## Fields
+| Field | Type |
+| --- | --- |
+| address | AztecAddress |
+| owner | AztecAddress |
+| randomness | Field |
+
+## Methods
+
+### new
+
+```rust
+AddressNote::new(address, owner);
+```
+
+#### Parameters
+| Name | Type |
+| --- | --- |
+| address | AztecAddress |
+| owner | AztecAddress |
+
+## Standalone Functions
+
+### compute_nullifier
+
+```rust
+compute_nullifier(self, context);
+```
+
+#### Parameters
+| Name | Type |
+| --- | --- |
+| self |  |
+| context | &mut PrivateContext |
+
+### compute_nullifier_without_context
+
+```rust
+compute_nullifier_without_context(self);
+```
+
+#### Parameters
+| Name | Type |
+| --- | --- |
+| self |  |
+
+### broadcast
+
+```rust
+broadcast(self, context, slot);
+```
+
+#### Parameters
+| Name | Type |
+| --- | --- |
+| self |  |
+| context | &mut PrivateContext |
+| slot | Field |
+
```

Comment:
Were these new reference docs autogenerated? Curious why it's showing as new.

Reply:
Yeah they should've been ignored, maybe they were cached, I'll remove them from git cache